peps/pep-8106.rst

```diff
@@ -0,0 +1,91 @@
+PEP: 8106
```

Comment:
Any particular reason for this number? By default we should follow the main numbering sequence, which I think means 735 or a nearby number at the moment.

Reply:
Yes, 735 should be next.

```console
$ pip install pepotron
...
$ pep next
Next available PEP: 735
```

Reply:
Oh, okay I thought 734 is the next available. Ok, I will change this.